process-engine/bpmn-studio
# 🐛Allow to add solution without identity

## Changes

1. This PR allows the user to add a solution which needs an identity. Now the user can add the identity after the solution was added.

## Issues

PR: #1766 

## How to test the changes

- start a process-engine with `allowGodToken` `false` config
- try to open the solution with the studio
- see that you can add the solution.
